app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
```

Comment:
Using a class variable `people` to store instances of `Person` can lead to unexpected behavior, especially if multiple instances are created or modified. Consider using an instance variable or another approach to manage person instances.